source/frontend/src/features/contacts/contact/create/components/address/Address.tsx

```diff
@@ -36,6 +36,12 @@ export const Address: React.FunctionComponent<React.PropsWithChildren<IAddressPr
   const { setFieldValue, values } = useFormikContext();
   const countryId = getIn(values, withNameSpace(namespace, 'countryId'));
 
+  const line1Count = !!getIn(values, withNameSpace(namespace, 'streetAddress1')) ? 1 : 0;
```

Comment:
This logic seems needlessly complex but rewriting it seems outside the scope of this hotfix.

Basically there are 2 major problems:
1) addressLines is an optional parameter that is not being set. I'm guessing the author meant to set that but forgot, so it always defaults to 1 even if multiple are present.
2) because this form displays as the data is loading, even if there are addressLines during the loading period there are not, which results in a useState in the useCounter hook being set to 1. useState initial values are not updated when they change so I had to add a useEffect to useCounter to allow the values of **initial** to propogate.